gir/pkgconfig/pkgconfig_test.go

```diff
@@ -2,42 +2,42 @@ package pkgconfig
 
 import (
 	"fmt"
-	"reflect"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestIncludeDirs(t *testing.T) {
-	tests := [][2][]string{
+	type testcase struct {
+		inPkgs     []string
+		expGIRDirs map[string]string
+	}
+	tests := []testcase{
```

Comment:
Nit: could you rewrite this in the style of

```go
tests := []struct{
    inPkgs        []string
    expectGIRDirs map[string]string
}{
    {
        {"gtk4"},
        {
            "gtk4": "/nix/store/niw855nnjgqbq2s0iqxrk9xs5mr10rz8-gtk4-4.2.1-dev/share/gir-1.0",
        },
    },
}
```

Also with the anonymous struct fields omitted in the value?

Reply:
Can't change `[]string{"gtk"}` to `{"gtk"}` or `map[string]string{…}` to `{…}`; but otherwise: ok.